tests/e2e/configs/sh-scripts/runFunctionalTests.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# shellcheck source=/dev/null
+
+validateParameters(){
+# Validate required parameters
+  if [ -z "$OCP_VERSION" ] || [ -z "$ARCH_VERSION" ] || [ -z "$TS_SELENIUM_BASE_URL" ]; then
+      echo "The ARCH_VERSION, OCP_VERSION or TS_SELENIUM_BASE_URL is not set!";
+      echo "Please, set all required environment variable parameters"
+      exit 1
+  fi
+}
+
+########################################
+############# Methods ##################
+########################################
+
+launchAPITests() {
+  export MOCHA_SUITE="APITest"
+  echo "MOCHA_SUITE = ${MOCHA_SUITE}"
+  export RP_LAUNCH_NAME="API tests suite"
+  echo "suites/$OCP_TYPE/$MOCHA_SUITE"
+  npm run driver-less-test
+}
+
+launchDynamicallyGeneratingAPITests() {
+  export MOCHA_SUITE="DynamicallyGeneratingAPITest"
+  export RP_LAUNCH_NAME="Application inbuilt DevWorkspaces API tests suite"
+  echo "MOCHA_SUITE = ${MOCHA_SUITE}"
+  echo "suites/$OCP_TYPE/$MOCHA_SUITE"
+  npm run delayed-test
+}
+
+launchUITests() {
+  export MOCHA_SUITE="UITest"
+  export RP_LAUNCH_NAME="UI tests suite"
+  echo "MOCHA_SUITE = ${MOCHA_SUITE}"
+  echo "suites/$OCP_TYPE/$MOCHA_SUITE"
+  npm run test
+}
+
+launchAllTests() {
+  validateParameters
+  initTestValues
+  echo ""
+  echo "Launching all tests for $TEST_ENVIRONMENT"
+  echo ""
+  launchDynamicallyGeneratingAPITests
+  launchAPITests
+  launchUITests
+}
+
+initTestValues() {
+  if [[ "$TS_SELENIUM_BASE_URL" =~ "airgap" ]]
```

Comment:
We have dedicated `IS_CLUSTER_DISCONNECTED` constant aimed to declare disconnected cluster.

Also, IBM Z / Power airgap OCP clusters doesn't contain `airgap` prefix in there host addresses.

So, what do you think about next condition?
```
if [[ -n "$IS_CLUSTER_DISCONNECTED" && "$IS_CLUSTER_DISCONNECTED" == "true" ]]
```

Reply:
how about initTestValues() {
  if [[ "$TS_SELENIUM_BASE_URL" =~ "airgap" || (-n "$IS_CLUSTER_DISCONNECTED" && "$IS_CLUSTER_DISCONNECTED" == "true") ]]
  then
     echo "Disconnected environment"
     export OCP_TYPE="disconnected-ocp"
  else
     echo "Online environment"
     export OCP_TYPE="online-ocp"
  fi

Reply:
okay, looks good